### PR TITLE
added initial GH template drafts - connect #380

### PR DESCRIPTION
**Pull request by altsang**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+### Submitting Pull Requests
+We use [Waffle.io](https://waffle.io/ExpressGateway/express-gateway) to track work on the task board. Waffle uses keywords just like Github.
+
+Please see the checklist below and thanks again for your contribution!
+
+- [ ] in description title please use "Connect #<Github Issue>" anywhere within the description
```
Comment (kevinswiber) [inline .github/PULL_REQUEST_TEMPLATE.md]: ~in description title~ please use "Connect #<Github Issue>" anywhere within the description

Comment (altsang) [reply .github/PULL_REQUEST_TEMPLATE.md]: 👍

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+### Submitting Pull Requests
+We use [Waffle.io](https://waffle.io/ExpressGateway/express-gateway) to track work on the task board. Waffle uses keywords just like Github.
+
+Please see the checklist below and thanks again for your contribution!
+
+- [ ] please use "Connect #<Github Issue>" anywhere within the description
+    example) "fixes bug with OAuth2 - connect #345"
```
Comment (kevinswiber) [inline .github/PULL_REQUEST_TEMPLATE.md]: Were you intending this to be on a new line?  You might have to add an extra linebreak.  It shows up like:

> please use "Connect #<Github Issue>" anywhere within the description example) "fixes bug with OAuth2 - connect #345"

Comment (altsang) [reply .github/PULL_REQUEST_TEMPLATE.md]: yeah new line but "inline" with the bullet indendation sans the bullet itself, thx

**Comment by kevinswiber** [comment]
LGTM

**Pull request merged**